examples/tomo/checks/check_axes_cone2d_fp.py

```diff
@@ -0,0 +1,98 @@
+"""Cone beam 2D example for checking that orientations are handled correctly.
+
+Due to differing axis conventions between ODL and the ray transform
+back-ends, a check is needed to confirm that the translation steps are
+done correctly.
+
+Both pairs of plots of ODL projections and NumPy axis sums should look
+similar in the sense that they should show the same features in the
```

Comment:
Not quite, the cone stuff is quite blurry, so we're looking for square-like stuff. We can also have new edges because of the depth-dependence in cone beam.